### ISSUE TITLE
Kysely on saavutettava kokonaisuudessaan

### ISSUE THREAD

**Issue by Nurou** [Devops-ohtuprojekti/DevOpsCSAOS#227]
Points: 13

- [x] Should be navigable with screen reader
- [x] Navigable by keyboard
- [x] Screen magnification
- [x] contrast
